freeipa/src/main/java/com/sequenceiq/freeipa/flow/stack/migration/AwsVariantMigrationFlowConfig.java

```diff
@@ -0,0 +1,77 @@
+package com.sequenceiq.freeipa.flow.stack.migration;
+
+import static com.sequenceiq.freeipa.flow.stack.migration.AwsVariantMigrationEvent.AWS_VARIANT_MIGRATION_FAILED_EVENT;
+import static com.sequenceiq.freeipa.flow.stack.migration.AwsVariantMigrationEvent.AWS_VARIANT_MIGRATION_FAIL_HANDLED_EVENT;
+import static com.sequenceiq.freeipa.flow.stack.migration.AwsVariantMigrationEvent.AWS_VARIANT_MIGRATION_FINALIZED_EVENT;
+import static com.sequenceiq.freeipa.flow.stack.migration.AwsVariantMigrationEvent.CHANGE_VARIANT_EVENT;
+import static com.sequenceiq.freeipa.flow.stack.migration.AwsVariantMigrationEvent.CREATE_RESOURCES_EVENT;
+import static com.sequenceiq.freeipa.flow.stack.migration.AwsVariantMigrationEvent.DELETE_CLOUD_FORMATION_EVENT;
+import static com.sequenceiq.freeipa.flow.stack.migration.AwsVariantMigrationFlowState.AWS_VARIANT_MIGRATION_FAILED_STATE;
+import static com.sequenceiq.freeipa.flow.stack.migration.AwsVariantMigrationFlowState.CHANGE_VARIANT_STATE;
+import static com.sequenceiq.freeipa.flow.stack.migration.AwsVariantMigrationFlowState.CREATE_RESOURCES_STATE;
+import static com.sequenceiq.freeipa.flow.stack.migration.AwsVariantMigrationFlowState.DELETE_CLOUD_FORMATION_STATE;
+import static com.sequenceiq.freeipa.flow.stack.migration.AwsVariantMigrationFlowState.FINAL_STATE;
+import static com.sequenceiq.freeipa.flow.stack.migration.AwsVariantMigrationFlowState.INIT_STATE;
+
+import java.util.List;
+
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.flow.core.config.AbstractFlowConfiguration;
+import com.sequenceiq.flow.core.config.AbstractFlowConfiguration.Transition.Builder;
+import com.sequenceiq.flow.core.config.RetryableFlowConfiguration;
+
+@Component
+public class AwsVariantMigrationFlowConfig extends AbstractFlowConfiguration<AwsVariantMigrationFlowState, AwsVariantMigrationEvent>
+        implements RetryableFlowConfiguration<AwsVariantMigrationEvent> {
+
+    private static final List<Transition<AwsVariantMigrationFlowState, AwsVariantMigrationEvent>> TRANSITIONS =
+            new Builder<AwsVariantMigrationFlowState, AwsVariantMigrationEvent>()
+                    .defaultFailureEvent(AWS_VARIANT_MIGRATION_FAILED_EVENT)
+                    .from(INIT_STATE).to(CREATE_RESOURCES_STATE).event(CREATE_RESOURCES_EVENT).defaultFailureEvent()
+                    .from(INIT_STATE).to(DELETE_CLOUD_FORMATION_STATE).event(DELETE_CLOUD_FORMATION_EVENT).defaultFailureEvent()
```

Comment:
Why do we need this? Is there a shortcut that skip CREATE_RESOURCES flow step?

Reply:
Just a leftover, I have remove it